lib/cmds/space_cmds/use.ts

```diff
@@ -94,11 +111,9 @@ async function spaceUse({ context, spaceId, header }) {
 
   await storeRuntimeConfig()
 
-  showSuccess(space, activeEnvironmentId)
+  showSuccess(space, activeEnvironmentId as string)
```

Comment:
Out of curiosity, why the cast?

Reply:
Context hasn't been migrated to TS yet. I'll fix that in another PR.
`showSuccess` needs the envId to print. So types for it is correct, context is what needs love.